python/desc/imsim/imSim.py

```diff
@@ -232,7 +236,20 @@ def sources_from_list(object_lines, obs_md, phot_params, file_name):
                 internal_rv[i_obj] = float(params[19])
             if params[i_gal_dust_model].lower() != 'none':
                 galactic_av[i_obj] = float(params[i_gal_dust_model+1])
-                galactic_rv[i_obj] =float(params[i_gal_dust_model+2])
+                galactic_rv[i_obj] = float(params[i_gal_dust_model+2])
+        elif (params[12].endswith('.fits') or params[12].endswith('.fits.gz')):
```

Comment:
This is OK. But I wanted to record the little figure I used to check this in case I want to go back and find it again:

```
blah.fits pixel_size _axis1 rot_angle  DUST_REST dust_pars_1(2) DUST_LAB dust_pars_2(2)
12          13                        14              15                 16/17               18                19/20

blah.fits pixel_size _axis1 rot_angle   None       DUST_LAB dust_pars_2(2)
12           13                        14              15             16              17/18
```

This doesn't exactly line up when rendered by GH for some reason.